src/www/diag_logs_settings.php

```diff
@@ -399,7 +399,8 @@ function check_everything() {
                       <div class="hidden" for="help_for_logfilesize">
                         <?=gettext("Logs are held in constant-size circular log files. This field controls how large each log file is, and thus how many entries may exist inside the log. By default this is approximately 500KB per log file, and there are nearly 20 such log files.") ?>
                         <br /><br />
-                        <?=gettext("NOTE: Log sizes are changed the next time a log file is cleared or deleted. To immediately increase the size of the log files, you must first save the options to set the size, then clear all logs using the \"Reset Log Files\" option farther down this page. "); ?>
+                        <strong><span class="text-danger"><?=gettext("NOTE:");?></span></strong>
```

Comment:
I'd like no more colours, no more uppercase for those leading words. It leads to exaggerated exposure of minor notes (unless they are not minor, but then they should neither be notes nor explanation text above is conceptually correct hiding important details in further notes). I'm also against splitting up the "Note:" and the rest of the text, creating multiple gettext calls, it loses context for translation.